appveyor.yml

```diff
@@ -42,7 +42,7 @@ before_test:
         Start-Service 'MySQL57'
         # Create nhibernate database (not handled by NHibernate.TestDatabaseSetup.dll)
         $env:MYSQL_PWD = 'Password12!'
-        & 'C:\Program Files\MySQL\MySQL Server 5.7\bin\mysql' -e 'CREATE DATABASE nhibernate;' --user=root
+        & 'C:\Program Files\MySQL\MySQL Server 5.7\bin\mysql' -e 'CREATE DATABASE nhibernate COLLATE utf8_general_ci;' --user=root
```

Comment:
@fredericDelaporte do you remember how we've fixed this (or similar) problem in the past?

Reply:
try with `CREATE DATABASE mydatabase CHARACTER SET utf8 COLLATE utf8_general_ci;`

Reply:
I'll try when this test run finishes.